x-pack/plugins/secops/public/components/timeline/body/renderers/suricata_row_renderer.tsx

```diff
@@ -61,6 +56,48 @@ const SuricataRow = styled.div`
   }
 `;
 
+const SuricataSignature = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-top 10px;
+`;
+
+const ButtonContainer = styled.div``;
+
+const Details = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 5px;
+  min-width: 340px;
+`;
+
+const LabelValuePairContainer = styled.div`
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+`;
+
+const Label = styled.div`
+  font-weight: bold;
+`;
+
+const Value = styled.div``;
```

Comment:
Same question about having empty styled components as above.

Reply:
I made this a regular `div`